tests/resetDB.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const Knex = require("knex");
+
+const AppConstants = require("../app-constants");
+const DB = require("../db/DB");
+const knexConfig = require("../db/knexfile");
+const {seed} = require("../db/seeds/test_subscribers");
+
+
+// (Re-)create DB connection at the beginning of each test suite
+beforeAll(async () => {
+  DB.createConnection();
+});
+
+
+beforeEach(async () => {
+  const knex = Knex(knexConfig[AppConstants.NODE_ENV]);
```

Comment:
Unrelated, but the more I see this line, the more I notice we always call it the same. What if ‘knexConfig’ just inspected the current `NODE_ENV` and only returned that config? Then we could just simplify to `Knex(knexConfig)` and not have to pass NODE_ENV each time.